dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java

```diff
@@ -683,12 +693,13 @@ private boolean createWorkflowsAndVersionsFromDockstoreYml(List<? extends Workfl
                         final List<YamlAuthor> yamlAuthors = wf.getAuthors();
 
                         Workflow workflow = createOrGetWorkflow(workflowType, repository, user, workflowName, wf, gitHubSourceCodeRepo);
+                        workflow.getUsers().addAll(otherUsers.stream().map(User::getId).map(userDAO::findById).toList()); // Because of Hibernate transactions, safest to lookup users again
```

Comment:
Just making sure I understand correctly, only users who are part of the push payload will be added to the workflow? And other users that are not in the payload but part of the same organization will have to use the Discover Existing Workflows button to see the workflow?

Reply:
Yes, correct. I touch upon it in the PR description -- my first attempt with this PR was to just add all of the repo's members to the workflow. But unfortunately, the GitHub app doesn't have permission view the repo/org members, so I fell back on this -- not complete, but hopefully better than what we have. If you're not one of the users in the event, you will have to Discover Workflows.

I think we eventually we should  have a scheduled job/task, leveraging the framework you're setting up for metrics, to sync GitHub permissions with entries.